software/managemeals.yml

```
@@ -0,0 +1,12 @@
+name: "ManageMeals"
+website_url: "https://managemeals.com/"
+source_code_url: "https://github.com/managemeals/manage-meals-web"
+description: "Manage recipes, import recipes by URL and organize them without any ads or unnecessary text."
+licenses:
+  - GPL-3.0
+platforms:
+  - Nodejs
+  - Docker
```

```suggestion
platforms:
  - Docker
```

The only tutorial I find on how to set up this project seems to be https://github.com/managemeals/manage-meals-api/wiki/Self%E2%80%90hosting, which only describes docker. Let me know if I missed something.

---

It's a Node.js app